frontend/Dockerfile.frontend

```diff
@@ -0,0 +1,6 @@
+FROM node:18
+WORKDIR /app
+COPY package*.json ./
```

Comment:
Confused why we're doing this copy separately instead of just copying all the files over in the first place